src/client.ts

```diff
@@ -427,6 +440,7 @@ export class Client {
 
   public async createStream(params: { stream: string; arguments?: CreateStreamArguments }): Promise<true> {
     this.logger.debug(`Create Stream...`)
+    this.filteringEnabled = lt(coerce(this.rabbitManagementVersion)!, REQUIRED_MANAGEMENT_VERSION) ? false : true
```

Comment:
I suggest to move this assignment in the connect callback, so it is performed as soon as the property is available